.github/workflows/ci_test-full.yml

```diff
@@ -135,6 +129,10 @@ jobs:
         path: Datasets
         key: pl-dataset
 
+    - name: Sanity check
+      run: |
+        python requirements/_check_avail_extras.py
```

Comment:
Same comment about pip list (L106)

Reply:
the same as above, we need to try to import these packages, if we make mistake in installation, related test are just skipped and we don't know about it...